applepaydomain/client.go

```diff
@@ -14,22 +14,24 @@ type Client struct {
 	Key string
 }
 
-// New POSTs new ApplePayDomains.
+// New creates a new ApplePayDomain.
```

Comment:
Minor, but I kind of think that we should try to use the "human style" form of "Apple Pay domain" in these comments (it's all intended for documentation). The function signature makes it obvious that a `stripe.ApplePayDomain` will be created.